upgrade incompatible

When I upgrade `html-webpack-plugin` from `v2.1.0` to `v2.6.0`. I get an error:

```bash
ERROR in ./~/html-webpack-plugin/lib/loader.js!./app/index.html
    Module not found: Error: Cannot resolve module 'lodash' in /Users/creeper/work/projects/react-base/app
     @ ./~/html-webpack-plugin/lib/loader.js!./app/index.html 1:8-25
```

I checked the source code, and found the reason:

```js
// lib/loader.js line26
return 'var _ = require("lodash"); module.exports = ' + template;
```

So it makes the `lodash` the **unneeded** dependency. And this change is from `v2.5.0`.

--- a/lib/loader.js
+++ b/lib/loader.js
@@ -22,5 +22,5 @@ module.exports = function (source) {
   // Use underscore for a minimalistic loader
   var options = loaderUtils.parseQuery(this.query);
   var template = _.template(source, options);
-  return 'var _ = require("lodash"); module.exports = ' + template;
+  return 'var _ = require("' + require.resolve('lodash') + '");module.exports = ' + template;
 };

